man/nvim.1

```diff
@@ -336,8 +336,22 @@ Print version information and exit.
 .El
 .Sh ENVIRONMENT
 .Bl -tag -width Fl
+.It Ev XDG_CONFIG_HOME
+Path to use for the user-local runtime directory, see
+.Sx FILES .
+Defaults to
+.Pa ~/.config
```

Comment:
You are correct, per [the spec](http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html):

> If $XDG_CONFIG_HOME is either not set or empty, a default equal to $HOME/.config should be used.

Then I don't know why there was discussion about where to put files on Windows: the spec commands that `~/.config` be used. This also means users can reliably predict the default on all systems and we don't need to qualify every help document and support request with "if you're on Windows [...] else [...]".